src/homepageExperience/graphics/CsvGraphic.tsx

```diff
@@ -0,0 +1,6 @@
+import React from 'react'
+import sampleCsv from 'assets/images/sample-csv.png'
+
+export const CsvGraphic = () => {
+  return <img style={{width: '100%', marginBottom: '48px'}} src={sampleCsv} />
+}
```

Comment:
So I know we talked about making this a component, but that was when I thought it was going to be an SVG. If this is going to be a .png that is imported, it's fine to just import the png and use the img tag directly, wrather than wrapping it in a component.

Either way, the `<img>` tag should have an [`alt`](https://blog.hubspot.com/marketing/image-alt-text) attribute for accessibililty.